components/pipedream_utils/actions/format-currency/format-currency.mjs

```
@@ -0,0 +1,73 @@
+      throw new ConfigurationError("**Invalid number** - please check your input.");
+    }
+    const numberString = formatNumber(integer, (decimal?.length > 1
+      ? decimal
+      : (decimal ?? "0") + "0"), currencyFormat.includes(",")
+      ? ","
+      : "");
+    result += numberString;
+    switch (currencyFormat.match(/¤+$/g)?.[0].length) {
+    default:
+      break;
+      // ¤¤ - ISO currency symbol: USD, BRL, etc.
```

Biome (1.9.4)

[error] The default clause should be the last switch clause.

The following case clause is here:

Regardless its position, the default clause is always executed when there is no match. To avoid confusion, the default clause should be the last switch clause.

(lint/suspicious/useDefaultSwitchClauseLast)